styles/styles.scss

```diff
@@ -32,30 +36,41 @@ h1.containerWrap {
 
 .mobileMenu {
     display: inline-block;
```

Comment:
Remove this property: `display: inline-block;` since you are using display flex.